update/Pos/Update/Configuration.hs

```diff
@@ -9,22 +10,31 @@ module Pos.Update.Configuration
        , withUpdateConfiguration
 
        , ourAppName
+       , currentSystemTag
```

Comment:
I think it's better not to export `currentSystemTag`, because it doesn't consider `UpdateConfiguration`, so it shouldn't be used.